### ISSUE TITLE
Windows KBDB "permission denied" after other person modified shared file

### ISSUE THREAD

**Issue by domdip** [keybase/client#4476]
Running on windows 10.  Drive letter is L: (in case that makes any difference).

I created a shard private folder with 1 other person, created a test file in it.  The other person modified the file.  Now I see "permission denied" to both public and private folders.

"keybase status" looks normal (I think).  I haven't tried rebooting.

my log id: a0e5efaa7eedf8c28cf1911c

**Comment by strib** [comment]
You are stuck in this state: "Refusing access to private directory while errors are present!"

@taruti: is this a dup of a known/fixed issue?  I don't see any actual failure that should block access, but seems like it could be related to keybase/kbfs#380 maybe?

**Comment by taruti** [comment]
No. That code uses `f.remoteStatus.ExtraFileName()` while that PR fixes  `f.remoteStatus.ExtraFileNameAndSize()` to behave like `f.remoteStatus.ExtraFileName()`.

However I think this issue was caused by `MDServerErrorThrottle{ProvisionedThroughputExceededException` which makes RemoteStatus enter into the error state and thus refusing access.

**Comment by strib** [comment]
@taruti: the previous throttle error was 7 minutes earlier -- is it a bug that it's still refusing access?

Also, that is a purely temporary error, and just indicates that the client should retry (which it does, automatically).  It seems like there's a bug here -- that error should not make its way into the error file.  Can you please take a look at it?

**Comment by taruti** [comment]
@strib, Thanks, got it.
